src/chatviewcommon.cpp

```diff
@@ -58,7 +58,7 @@ QString ChatViewCommon::getMucNickColor(const QString &nick, bool isSelf)
             /* Hash-driven colors */
             quint32 hash = qHash(nickwoun); // almost unique hash
             QList<QColor> &_palette = generatePalette();
-            return _palette.at(hash % _palette.size()).name();
+            return _palette.at(int(hash) % _palette.size()).name();
```

Comment:
instead cast the result of %, otherwise you are in risk of getting a negative index